pkgs/development/python-modules/alot/default.nix

```
@@ -16,18 +16,9 @@ buildPythonPackage rec {
     owner = "pazz";
     repo = "alot";
     rev = version;
-    sha256 = "sha256-WUwOJcq8JE7YO8sFeZwYikCRhpufO0pL6MKu54ZYsHI=";
+    hash = "sha256:0s94m17yph1gq9f2svipb3bbwbw1s4j3zf2xkg5h91006v8286r6";
```

most standard way in nixpkgs
```suggestion
    sha256 = "0s94m17yph1gq9f2svipb3bbwbw1s4j3zf2xkg5h91006v8286r6";
```

---

Oh, really? Curiously [someone on IRC](https://logs.nix.samueldr.com/nixos/2020-05-14#3458998;) implied that would not be the best way to go, but no worries I'll change it.

---

The `hash` form is for SRI-style hashes (see the [Nix 2.2 release notes](https://nixos.org/nix/manual/#ssec-relnotes-2.2)), is the default output of new enough Nix, and will be preferred in the future, so it's fine to use.

---

oh interesting, sorry, didn't know